lib/savings_account.rb

```diff
@@ -0,0 +1,23 @@
+module Bank
+	class SavingsAccount < Account 
+		def initialize(account_id, balance, date_opened)
+			super
+			@min_balance = 1000
+		end
+
+		def withdraw(withdraw_amount)
+			transaction_fee = 200
+			puts "Your balance was #{@balance}."
+
+			if @balance - withdraw_amount <= @min_balance
+				puts "Don't drop below 10 dollars w/ transaction fee."
```

Comment:
It would be nice to use you `@min_balance` in this string to ensure that if that variable was changed, the error message would also be chaqnged.